src/astra_driver.cpp

```diff
@@ -79,6 +79,10 @@ AstraDriver::AstraDriver(rclcpp::node::Node::SharedPtr& n, rclcpp::node::Node::S
   z_scaling_ = 1.0;
   z_offset_mm_ = 0;
 
+  AstraVideoMode color_video_mode{width, height, framerate, PIXEL_FORMAT_RGB888};
+
+  setColorVideoMode(color_video_mode);
```

Comment:
No worries.  I should have thought of that.  I just added another commit here which allows us to change the depth resolution, as well as 1MM or 100UM accuracy.  While I was in there, I also removed the check in main() for the resolution, which was kind of silly.  Now we just let the driver tell us whether it is supported.  I still don't love it (as you could ask for a resolution that can't possibly work), but as this is temporary code, I don't think it matters too much.